src/script_opt/ProfileFunc.cc

```diff
@@ -867,11 +867,11 @@ void ProfileFuncs::ComputeProfileHash(std::shared_ptr<ProfileFunc> pf) {
         h = merge_p_hashes(h, p_hash(ov[i]->Name()));
 
     h = merge_p_hashes(h, p_hash("stmts"));
-    for ( auto i : pf->Stmts() )
+    for ( auto& i : pf->Stmts() )
```

Comment:
```suggestion
    for ( const auto& i : pf->Stmts() )
```

Reply:
and re the first one: oops, thanks for catching!